src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
 ---
-title: Get started with server-side Dart
-description: Get Dart and run a Dart app.
+title: "Get started: command-line & server apps"
+description: Get Dart, run and compile a small app
+toc: true
 nextpage:
   url: /tutorials/server/cmdline
   title: Write command-line apps
 prevpage:
   url: /tutorials/server
-  title: Server-side Dart tutorials
+  title: Dart command-line and server tutorials
 ---
 
-<div class="mini-toc" markdown="1">
-  <h4>What's the point?</h4>
-
-  * The Dart SDK has development tools and libraries.
-  * Use an IDE or a code editor to create your app.
-  * All Dart apps have a `main()` function.
-  * Dart supports top-level functions.
+Follow these steps to start using Dart to develop command-line & server apps.
+First you’ll play with Dart in your browser, no download required. Then you’ll
+install Dart and run a small program with the Dart VM. Finally, you'll
+AOT-compile that program to native machine code.
+
+## 1. Play with a command-line app in DartPad
+
+With DartPad you can experiment with the Dart language and APIs,
+no download necessary.
+
+For example, here's an embedded DartPad that lets you play with
+the code for a small Hello World program.
+Click run {% asset red-run.png alt="" %} to run the app;
+the console output appears beneath the code.
+Try editing the source code—perhaps you'd like to change the greeting
+to use another language. To get the full DartPad experience,
+which includes the web UI that the app produces,
+<a href="https://dartpad.dartlang.org/27e044ec9e2957d9c5c7062871ce8bf3"
+   target="_blank">open the example at dartpad.dartlang.org.</a>
+
+<iframe
+    src="{{site.custom.dartpad.embed-inline-prefix}}?id=27e044ec9e2957d9c5c7062871ce8bf3&verticalRatio=70"
+    width="100%"
+    height="200px"
+    style="border: 1px solid #ccc;">
+</iframe>
+
+More information:
+
+* [DartPad documentation][]
+* [Dart language tour][]
+* [Dart library tour][]
+
+## 2. Install Dart
+
```

Comment:
We need an intro here. The one in the web page seems good enough:

Once you’re ready to move beyond DartPad and develop real apps, you need the Dart SDK.